### DISCUSSION TITLE
The SteamGridDB page for custom assets just got approved

### DISCUSSION THREAD

**Discussion by SecularSteve** [MissLav/LibreQuake#150] [Show and tell]
https://www.steamgriddb.com/game/5461287

![3eeae778e639024c637a960e63eece9c](https://github.com/user-attachments/assets/186f9e4e-ce52-4c71-8523-732a22ef9a94)
![14e8e0b673b76a21e0e00d3df738d8c5](https://github.com/user-attachments/assets/3132bf33-7a11-4c42-a1b1-88dcf1af9e19)

Made these two myself using the provided assets from here. The rest I added in straight from the Git.

The only thing I didn't add in is the rest of the Logos, and the varieties thereof. Figures it probably needs to be discussed first. But that's just me not trying to deliberately overstep boundaries.

What do you think?

**Comment by lavenderdotpet** [comment]
oh id prob recommend using these instead 
![image](https://github.com/user-attachments/assets/0ede98ea-0dca-451b-8c5d-2b16e9d81768)

ive been thinking of wiping up my own might do that rn

also idk if one is able to change this but lq got started in 2019 
![image](https://github.com/user-attachments/assets/2429ba57-7448-4f8b-881f-91af3689e92f)

**Reply by SecularSteve** [reply]
I'll make the request to fix it on Discord. They're usually quick about it.

Idk who thought it was 2022, but I didn't include the date in my request. I simply assumed they'd look into it.

**Reply by SecularSteve** [reply]
UPDATE:
It was deliberate because this was the first public release of the game:
https://github.com/lavenderdotpet/LibreQuake/tree/v0.01-test

More specifically, on the 1st of July 2022 (or was it the 7th of January, American Date formats confuse me).

Do you want it to remain 2022 in lieu of the above, or do you want it to include the earlier semi-playable testing versions all the way back to 2019?

**Reply by lavenderdotpet** [reply]
could the date just be removed that would be best

**Reply by SecularSteve** [reply]
They did say the easier step is to just switch it to 2019 if that's what you really want, and gave me this sheet on how they base their dating convention for games:

![image](https://github.com/user-attachments/assets/b10d7db0-2df1-4297-b068-d166171c2141)

I'll just let them know if that's fine with you.

**Reply by SecularSteve** [reply]
done...

![Screenshot 2024-08-15 at 00-03-54 LibreQuake - SteamGridDB](https://github.com/user-attachments/assets/23fd06f8-4efb-498c-b234-d50d0bd39ee9)

**Reply by lavenderdotpet** [reply]
~~ig 2019 then~~
edit ah nvm didnt see ur response 

also heres a show case of the wip mockup 
border is very wip just used the one from the svg icon i made
![image](https://github.com/user-attachments/assets/1464b3a3-bbb2-4e69-bfeb-27162c018570)

**Reply by SecularSteve** [reply]
Nice...
Just keep these in mind for the proportions (in pixels):

Capsules/Grids: 460x215, 920x430 (Horizontal), 600x900 (Vertical)
Hero Images: 1920x620, 3840x1240
File Type: PNG, JPEG, Animated WebP
Max. Capsule Size: 15MB for static, 48MB for animated.
Max. Hero Size: 24MB for static, 48MB for animated.

**Reply by lavenderdotpet** [reply]
yeah fixing the 600x900 rn

**Comment by SecularSteve** [comment]
UPDATE:

Just found out theoutrider's uploaded a whole bunch of their own custom assets nearly a month ago, a ton of varieties at that:

https://www.steamgriddb.com/profile/76561197968125041

https://www.steamgriddb.com/game/5461287/grids
https://www.steamgriddb.com/game/5461287/heroes

Your take on it?

**Reply by fpiesche** [reply]
Hello, it me :3

My artwork here is based on some wallpapers one of the texture artists shared on the LQ discord, fwiw. SGDB's artwork is entirely community contributed so anyone could upload their own custom cover artwork; I'd mostly done this as the art you'd created did not include square grids which are used by a number of the community regulars over there. Plus, the wallpapers are quite nice so I figured it'd be nice to have cover/banner art based on them also :)

**Reply by SecularSteve** [reply]
Very nice 👍

Just don't forget the Logos and Icons.
